integration_test/conftest.py

```diff
@@ -212,7 +238,6 @@ def log_file_in_tmpdir(tmpdir):
     os.chdir(old_workdir)
 
 
-@pytest.fixture
```

Comment:
I removed it because you're not allowed to call a fixture explicitly, and I needed to call this in another function (`run_manager_with_actors`). Since other files no longer need access to this, the easiest way to solve it was by removing the fixture decorator.

Reply:
Ah, perfect!